### ISSUE TITLE
Combine animations

### ISSUE THREAD

**Issue by David-Klemenc** [GregorGrasselli/pendula-animation#1]
```elm
drawPendulum : (Float, Float) -> Pendulum -> List (Svg.Svg Msg)
drawPendulum ((originX, originY) as stringOrigin) pendulum =
    let d = distanceToWeightCenter pendulum
        (circleX, circleY) = getPosition stringOrigin pendulum.currentAngle d
        (stringX, stringY) = getPosition stringOrigin pendulum.currentAngle pendulum.stringLength
    in
        [ circle [ cx circleX
                 , cy circleY
                 , r (String.fromFloat pendulum.weightRadius)
                 , fill (circleFill pendulum.marked)
                 , stroke "black"
                 , strokeWidth "5"
                 ] []
        , line [ x1 stringX
               , x2 (String.fromFloat originX)
               , y1 stringY, y2 (String.fromFloat originY)
               , stroke "black"
               , strokeWidth "3"
               ] []
        ]
```
Both the line and circle can be animated together ... can you figure out how?
(imagine that instead of a circle you would have to draw an elephant or a clock - calculating properties for each clock dial or elephant tusk would be a lot of work ...)

HINT: use `g` to wrap things together.

**Comment by GregorGrasselli** [comment]
Thank you for the hint. Will fix.

**Comment by GregorGrasselli** [comment]
Should be solved by PR #2
